tests/yam/validate/validate_installed_patterns.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate installed patterns against the list of additional patterns
+# selected to be installed during installation.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use utils 'zypper_call';
+use testapi;
+
+sub run {
+    select_console 'root-console';
+
+    my @pattern_list = split(/,/, get_var('PATTERNS'));
+
+    zypper_call("search -t pattern");
+    zypper_call("search -i -t pattern");
```

Comment:
rebase after merge https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21394/files#diff-0c3c7b906c2b6f8ca97690eb7bd8b81cb039736b0ca72c717a366f0eb820d77aR21
so we can apply current formatting here.

Reply:
rebased.